client/src/css/style.css

```diff
@@ -219,6 +229,10 @@ body {
     -webkit-backdrop-filter: blur(16px);
     overflow: hidden;
     margin: 0 auto;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 1fr 1fr;
+    container: calculator / size;
```

Comment:
how does this line work? where are the values ​​for calculator and size defined?